lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py

```diff
@@ -169,7 +169,6 @@ def variantOrNewPrim(buttonChecked):
     variantSelected = cmds.radioButtonGrp('variantRadioButton', query=True, select=1)
     cmds.rowLayout('usdCacheVariantSetRow', edit=True, enable=variantSelected)
     cmds.rowLayout('usdCacheVariantNameRow', edit=True, enable=variantSelected)
-    cmds.textFieldGrp('primNameText', edit=True, enable=not variantSelected)
```

Comment:
According to the JIRA item there was a mistake in the original design. This field should be enabled all the time.